docs/lab/at-home-phlebotomy/communications.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: "Communications"
+---
+
+Communications with the patient are carried in two different ways for At-Home Phlebotomy orders, being them
+email and SMS messages. There are three distinct communications options:
```

Comment:
Communication for patients is done via email or SMS, for At-Home Phlebotomy orders. Our customers have the following options when setting this up: